api/v1alpha1/constants.go

```diff
@@ -6,8 +6,14 @@ package v1alpha1
 const (
 	// OperationAnnotation indicates which operation should be performed outside the current spec definition flow.
 	OperationAnnotation = "metal.ironcore.dev/operation"
-	// OperationAnnotationIgnore skips the reconciliation of a resource if set to true.
+	// PropagatedOperationAnnotation indicates which operation should be performed outside the current spec definition flow.
+	PropagatedOperationAnnotation = "metal.ironcore.dev/operation-propogated"
+	// OperationAnnotationIgnore skips the reconciliation of a resource if OperationAnnotation is set to this.
 	OperationAnnotationIgnore = "ignore"
+	// OperationAnnotationIgnoreChild skips the reconciliation of a resource's Child if OperationAnnotation is set to this.
+	OperationAnnotationIgnoreChild = "ignore-child"
+	// PropagatedOperationAnnotationIgnored skips the reconciliation of a resource if PropagatedOperationAnnotation is set to this.
+	PropagatedOperationAnnotationIgnored = "propagated-ignore"
```

Comment:
Usage of this value in full reads `metal.ironcore.dev/operation-propogated: propagated-ignore`. Maybe we can simply reuse `ignore` here?